requirements.txt

```diff
@@ -1,21 +1,105 @@
 asgiref==3.6.0
 attrs==22.2.0
+
+backcall==0.2.0
```

Comment:
As we are not adding new packages for this task. Please remove the packages as they are causing the problems